test-utils/testlib/src/alphanet_utils.rs

```diff
@@ -27,15 +28,265 @@ pub fn configure_chain_spec() -> ChainSpec {
     read_or_default_chain_spec(&Some(d))
 }
 
-pub struct Node {
-    pub client: Arc<Client>,
+pub enum NodeType {
+    ThreadNode,
+    ProcessNode,
+}
+
+pub enum ProcessNodeState {
+    Stopped,
+    Running(Child),
+}
+
+pub struct NodeConfig {
     pub node_info: PeerInfo,
     pub client_cfg: ClientConfig,
     pub network_cfg: NetworkConfig,
     pub rpc_cfg: RPCConfig,
+    pub peer_id_seed: u32,
```

Comment:
`peer_id_seed` is already available in the `NetworkConfig`, no need to pass it separately.

Reply:
the id in NetworkConfig is the hash of the seed